RockWeb/Blocks/Finance/TransactionEntry.ascx.cs

```diff
@@ -291,6 +299,10 @@ protected override void OnInit( EventArgs e )
             lSuccessHeader.Text = GetAttributeValue( "SuccessHeader" ).ResolveMergeFields( configValues );
             lSuccessFooter.Text = GetAttributeValue( "SuccessFooter" ).ResolveMergeFields( configValues );
 
+            // Determine account campus context mode
+            _accountCampusContext = GetAttributeValue( "AccountCampusContext" ).AsType<int>();
+            _currentCampusContext = RockPage.GetCurrentContext( EntityTypeCache.Read( typeof( Campus ) ) ).Id;
```

Comment:
@treyhendon it seems like this might cause a NULL exception if a Campus Context is not set.